src/Client/ClientBase.cpp

```diff
@@ -3211,6 +3211,9 @@ void ClientBase::runInteractive()
 
 bool ClientBase::processMultiQueryFromFile(const String & file_name)
 {
+    if (global_context->getApplicationType() == Context::ApplicationType::EMBEDDED_CLIENT)
```

Comment:
Likely this could also exploit the server:
- `output_format_schema`
- `format_schema`
- `input_format_record_errors_file_path`
- and likely some other format settings

- `--server_logs_file`
- `--client_logs_file`
- `--buzz-house-config`
- `--history_file`
- `--config`